pkg/infinispan/cluster.go

```diff
@@ -275,3 +241,58 @@ func (c Cluster) GetCacheManagerInfo(cacheManagerName, podName string) (map[stri
 	}
 	return info, nil
 }
+
+func (c Cluster) GetLoggers(podName string) (map[string]string, error) {
+	rsp, err, reason := c.Client.Get(podName, consts.ServerHTTPLoggersPath, nil)
+	if err := validateResponse(rsp, reason, err, "getting cluster loggers", http.StatusOK); err != nil {
+		return nil, err
+	}
+
+	defer rsp.Body.Close()
+	var loggers []Logger
+	if err := json.NewDecoder(rsp.Body).Decode(&loggers); err != nil {
+		return nil, fmt.Errorf("unable to decode: %v", err)
+	}
+	lm := make(map[string]string)
+	for _, logger := range loggers {
+		if logger.Name != "" {
+			lm[logger.Name] = logger.Level
```

Comment:
We can do `strings.ToLower(logger.Level)` here so that it's not necessary in the consuming methods.

Reply:
Yes, will fix